fedocal/fedocallib/__init__.py

```diff
@@ -713,8 +717,19 @@ def add_meeting_to_vcal(ical, meeting):
                 until=meeting.recursion_ends))
         entry.rruleset = newrule
 
+    if enable_reminder:
+        for trigger in (
+                timedelta(**reminder) for reminder in reminders):
```

Comment:
So this means that by default we get 3 reminders for a single meeting?
Seems a little generous to me :-)